Objects/unicodeobject.c

```diff
@@ -11611,6 +11611,25 @@ unicode_index(PyObject *self, PyObject *args)
     return PyLong_FromSsize_t(result);
 }
 
+/*[clinic input]
+str.isascii as unicode_isascii
+
+Return True if all characters in the string are ASCII, False otherwise.
```

Comment:
nitpick, maybe copy from the doc: "Return true if the string is empty or all characters in the string are ASCII," rather than "Empty string is ASCII too." below.

Reply:
"Return true if the string is empty or all characters in the string are ASCII, False otherwise." overs 80 columns.
And clinic show error when I wrap the line.

All other docstrings in unicodeobject has short (<80) summaries.

Reply:
Oh wow, that's a nasty issue. Ignore my comment and leave the docstring as it is ;-)